pkg/protocol/request.go

```diff
@@ -347,7 +347,7 @@ func (req *Request) SwapBody(body []byte) []byte {
 func (req *Request) CopyTo(dst *Request) {
 	req.CopyToSkipBody(dst)
 	if req.bodyRaw != nil {
-		dst.bodyRaw = req.bodyRaw
+		dst.bodyRaw = append(dst.bodyRaw, req.bodyRaw...)
```

Comment:
`append(dst.bodyRaw[:0],req.bodyRaw...) ` is better? In case exist data in dst.bodyRaw?